aospy/calc.py

```diff
@@ -767,3 +767,19 @@ def load(self, dtype_out_time, dtype_out_vert=False, region=False,
         if plot_units:
             data = self.var.to_plot_units(data, dtype_vert=dtype_out_vert)
         return data
+
+    def addAttrs(self, data):
+        if (isinstance(data, xr.DataArray)):
+            self.addAttrsDataArray(data)
+        else:
+            for name, da in data.data_vars.items():
+                self.addAttrsDataArray(da)
+
+    def addAttrsDataArray(self, data):
+        if self.var.units != '':
+            if self.var.dtype_out_vert == 'vert_int':
+                data.attrs['units'] = '(vertical integral of {0}): {0} kg m^-2)'.format(self.var.units)
+            else:
+                data.attrs['units'] = self.var.units
+        if self.var.description != '':
+            data.attrs['description'] = self.var.description
```

Comment:
nit: make sure to add a newline at the end of the file (that's what the red "no" symbol at the end of this line on GitHub signifies).

Reply:
I'm actually inclined to go ahead and write the description to the file even if it's blank.  That makes the logic simpler and also makes the netCDF files outputted by aospy more homogeneous.  I.e. once this is merged, users can always rely on there being a 'description' attr.